examples/getting-started/example1/main.cpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <exception>
+#include <fstream>
+#include <iostream>
+
+#include <powsybl/PowsyblException.hpp>
+#include <powsybl/iidm/Network.hpp>
+
+int main(int /*argc*/, char** argv) {
+    try {
+        std::ifstream inputStream(argv[1]);
```

Comment:
To  be cleaner, you should ensure the length of the argv array to avoid out-of-bound error.